test/extended/apibinding/apibinding.go

```diff
@@ -0,0 +1,44 @@
+package apibinding
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[sig-apibinding]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-apibinding")
+	)
+
+	g.It("Author:pewang-Critical-[Smoke] Create an apibinding attach a workspace to the exist shared compute should work well", func() {
```

Comment:
How about paying more attention to normal syntax, such syntax issues seem often ignored : ) ?
Here, your statement in short is `Create xxx attach xxx should work`. Normal syntax is `Creating xxx should work`. In addition, for `Create an apibinding attach a workspace`, `Create` and `attach` both are verb, your format `verb1 xxx verb2 xxx should xxx` do not have connective words between two verbs.
s/Create an apibinding attach a workspace/Creating an apibinding to attach a workspace/
s/exist/existing/

Same for other places:
g.By("# Create an apibinding attach a workspace to the exist shared compute")
et al

Reply:
Sure, thanks sorry for that. I'll paying more attention about the normal syntax for the steps descriptions and comments of functions.

Reply:
@Phaow I think test case name should be changed to "Create workload using shared compute provided by ACM should work well". The above is very confusing to be honest, thanks !!

Reply:
@kasturinarra Sure, thanks for point it out. I'll change the scenario to an exist case which ke suggested.

Reply:
@Phaow what is the exist case that ke suggested ? could you please help me understand ? Also the scenario title is confusing. Can we rename it to "Create and schedule workload using shared compute provided by ACM should work well" if no one else has any objections ?